FFXIVClientStructs/FFXIV/Client/System/Framework/Framework.cs

```diff
@@ -50,6 +51,14 @@ public unsafe partial struct Framework {
 
     [FieldOffset(0x2BF0)] public GameVersion GameVersion;
 
+    /// <summary>
+    /// Access the Steam API wrapper/interface.
+    /// </summary>
+    /// <remarks>
+    /// The struct backed by this API should not be considered stable. If you use this, you are signing up for API breakage.
+    /// </remarks>
+    [FieldOffset(0x35B8)] public SteamApi SteamApi;
```

Comment:
I'm pretty sure this is supposed to be a pointer (see `ffxiv_dx11.exe+95A1D`).
```suggestion
    [FieldOffset(0x35B8)] public SteamApi* SteamApi;
```

Reply:
yeah, it is. i was sleep deprived last night.